frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -133,6 +138,33 @@ export const GRAPH_DATA_SETTINGS = {
     dashboard: false,
     useRawSeries: true,
   },
+  series_order: {
```

Comment:
I briefly glace at all the settings names within this file and seems like they have some common prefixes. e.g.
- graph
- line
- scatter

And seems generic settings always has `graph` prefix. Following this principle, what do you think about renaming this `graph.series_order` to make it similar to other settings names?

Reply:
Makes sense